scipy/stats/_distn_infrastructure.py

```diff
@@ -3913,6 +3911,175 @@ def _get_support(self, *args):
         """
         return self.a, self.b
 
+    def _pmf(self, x):
+        return rv_count._pmf(self, x)
+
+    def _cdf(self, x):
+        return rv_count._cdf(self, x)
+
+    def _ppf(self, q):
+        return rv_count._ppf(self, q)
+
+    def _rvs(self, size=None, random_state=None):
+        return rv_count._rvs(self, size, random_state)
+
+    def _entropy(self):
+        return rv_count._entropy(self)
+
+    def generic_moment(self, n):
+        return rv_count.generic_moment(self, n)
+
+    def _expect(self, fun, lb, ub, *args, **kwds):
+        return rv_count._expect(self, fun, lb, ub, *args, **kwds)
+
+
+class rv_count(rv_discrete):
+    """A discrete random variable defined by support points and probabilities.
+
+    Parameters
+    ----------
+    xk : array_like
+        Support points.
+    pk : array_like
+        The probabilities, corresponding to `xk`
+    name : str, optional
+        The name of the instance. This string is used to construct the default
+        example for distributions.
+    longname : str, optional
+        This string is used as part of the first line of the docstring returned
+        when a subclass has no docstring of its own. Note: `longname` exists
+        for backwards compatibility, do not use for new subclasses.
+    seed : {None, int, `numpy.random.Generator`, `numpy.random.RandomState`}, optional
```

Comment:
I just had another PR merge with a lint failure and I saw the issue in another PR. I don't want that to happen here, so I'm changing this to how it appears in some other stats functions.
```suggestion
    seed : {None, int, `numpy.random.Generator`,
            `numpy.random.RandomState`}, optional

```